.kokoro/presubmit/java8-samples.cfg

```diff
@@ -26,7 +32,3 @@ env_vars: {
   value: "java-docs-samples-service-account"
 }
 
-env_vars: {
```

Comment:
Please ignore deletion, I do not know why git thinks we moved this file from `.../nightly/samples/common.cfg` to `.../presubmit/java8-samples.cfg`